server/router/src/test/java/org/infinispan/server/router/utils/HotRodClientTestingUtil.java

```diff
@@ -0,0 +1,52 @@
+package org.infinispan.server.router.utils;
+
+import org.infinispan.client.hotrod.RemoteCacheManager;
+import org.infinispan.client.hotrod.configuration.ConfigurationBuilder;
+
+import java.net.InetAddress;
+
+public enum HotRodClientTestingUtil {
```

Comment:
`HotRodClientTestingUtil` assumes in many methods that it talks to the `HotRodServer` so it gets some configuration bits from it. As you can see the `HotRodClientTestingUtil` from the router module plays more with the configuration. Putting everything together will result in spaghetti testing util which I would like to avoid.